lib/Controller/RoomController.php

```diff
@@ -476,7 +477,7 @@ protected function formatRoomV1(Room $room, ?Participant $currentParticipant): a
 		$numActiveGuests = 0;
 		$cleanGuests = false;
 		$participantList = [];
-		$participants = $this->participantService->getParticipantsForRoom($room);
+		$participants = $this->participantService->getParticipantsForRoom($room, true); // FIXME NEEDS the session but can potentially kill APIv1?
```

Comment:
I think we can just not keep this alive anymore. we can drop this when mobile clients all speak APIv3 by default and just fallback to APIv1 when no APIv3 is provided. that should work for iOS already and Android is on the way.

Reply:
this is still the question what to do.
I would introduce a v4 with the current changes that take place to make clear that only Talk sessions of the current PHP session are returned for the current user and basically would like to kill all older room APIs, as their behaviour is inconsistent to how it was before and we can't keep it backwards compatible.